velox/exec/tests/TableWriteTest.cpp

```diff
@@ -108,6 +108,23 @@ std::function<PlanNodePtr(std::string, PlanNodePtr)> addTableWriter(
   };
 }
 
+RowTypePtr getScanOutput(
```

Comment:
Why is this utility function called ScanOutput?

Reply:
Updated to `getNonPartitionsColumns`.